core/jazz_ui/Jenkinsfile_Platform

```diff
@@ -107,6 +107,18 @@ node ()  {
       def importUICommand = 'sed -i s/{INSTALLER_VARS}/' + uiConfig + '/g ./src/environments/environment.oss.ts'
       sh importUICommand
 
+      sh "sed -i 's/AWS API Gateway/\"AWS API Gateway\"/g' ./src/environments/environment.oss.ts"
+      sh "sed -i 's/aws_apigateway/\"aws_apigateway\"/g' ./src/environments/environment.oss.ts"
+      sh "sed -i 's/APIGEE/\"APIGEE\"/g' ./src/environments/environment.oss.ts"
```

Comment:
This should be called GCP Apigee and not just Apigee

Reply:
Just the label is named apigee, the value used in UI code is still gcp_apigee.
Should we change the label as well?
We ll have to make changes in all the places where this flag is used

Reply:
ideally yes

Reply:
fixed this

Reply:
to be merged with the installer side PR401